packages/catana/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
+pkgrel=1
+pkgdesc='CATANA filters your wordlist according to the specified password policy.'
+groups=('blackarch' 'blackarch-misc')
+arch=('any')
+url='https://github.com/D3vil0per/catana'
+license=('GNU General Public License v3.0')
```

Comment:
It's not how it works. https://wiki.archlinux.org/title/PKGBUILD#license

Reply:
Commit a9867a8b3d55ce47cf7cf344bffc2838f7856422